src/axis.ts

```diff
@@ -0,0 +1,48 @@
+import * as d3 from "d3";
+import { Pareto} from "./pareto";
+
+ export async function drawLine(pareto: Pareto) {
```

Comment:
I think naming here is misleading, since this draws the axes, drawAxes is more appropriate